README.rst

```diff
@@ -27,11 +27,11 @@ Please cite the following DOI if you are using processed LCOGT data.
 
 Installation
 ------------
-BANZAI can be installed in the usual way, by running
+BANZAI can be installed using pip, by running from the top-level directory containing `setup.py`.
 
 .. code-block:: bash
-
-    python setup.py install
+    pip install --upgrade pip
```

Comment:
This feels like bad form to have this in the docs.

Reply:
Add to setup_requires in setup.cfg